### PR TITLE
Add Bourbon as a dependency

### PR DESCRIPTION
**Pull request by tysongach**
Neat 1.8 was just released, which removed Bourbon as a dependency.

Up until now, Neat brought in Bourbon (which Administrate needs). But
that was fragile and now that Neat doesn't explicitly declare Bourbon as
a dependency, this nested dependency has fallen apart.

### CODE REVIEW

**Comment by tysongach** [comment]
Looks like the appraisal for Bourbon 5 is breaking the build.

I’m unsure and confused why we have the appraisal; not sure it’s providing any value? Upgrading to Bourbon 5 will force some CSS refactoring and supporting both Bourbon 4 _and_ 5 at the same time will not be possible.

**Comment by jalada** [comment]
My workaround for now is just to add Bourbon as a dependency in my Gemfile.

**Comment by teoljungberg** [comment]
@tysongach I agree, I removed the Appraisal for Bourbon 5 in this PR.

File: Gemfile [hunk 1]
```diff
@@ -3,6 +3,7 @@ source 'https://rubygems.org'
 gemspec
 
 gem "administrate-field-image"
+gem "bourbon"
```
Comment (tysongach) [inline Gemfile]: @teoljungberg Shall we lock this down to `4.x`, `, "~> 4.2"`? Once 5 ships, it will bust.

Comment (teoljungberg) [inline Gemfile]: Ah, good point! Done in 273373b8290c7780ed284eb65c7b207cb339ac8e

**Comment by tysongach** [comment]
One more comment…

Otherwise 👍 👍 👍 👍 👍 👍 👍

File: gemfiles/sass_3_4.gemfile [hunk 2]
```diff
@@ -2,9 +2,8 @@
 
 source "https://rubygems.org"
 
-ruby "2.2.3"
-
 gem "administrate-field-image"
+gem "bourbon"
```
Comment (teoljungberg) [inline gemfiles/sass_3_4.gemfile]: Gah, I'm sloppy today. Thanks for being my eyes, Tyson!

**Comment by tysongach** [comment]
Thanks, @teoljungberg!

**Pull request merged**